src/main/java/io/cdap/plugin/sendgrid/SendgridPostAction.java

```diff
@@ -53,37 +58,88 @@
    * Plugin config for Sendgrid post action.
    */
   public static class Config extends ConditionConfig {
+    private static final String FROM = "from";
+    private static final String TO = "to";
+    private static final String SUBJECT = "subject";
+    private static final String CONTENT = "content";
+    private static final String API_KEY = "apiKey";
+    private static final String INCLUDE_WORKFLOW_TOKEN = "includeWorkflowToken";
+
     @Description("The address to send the email from.")
-    @Name("from")
+    @Name(FROM)
     @Macro
     private String from;
 
     @Description("The subject of the email")
-    @Name("subject")
+    @Name(SUBJECT)
     @Macro
     private String subject;
 
     @Description("The address to send the email to.")
-    @Name("to")
+    @Name(TO)
     @Macro
     private String to;
 
     @Description("The SendGrid API Key. After logging into your Sendgrid account, you can create a key in the " +
       "API keys section of the Settings page.")
-    @Name("apiKey")
+    @Name(API_KEY)
     @Macro
     private String apiKey;
 
     @Description("Optional content of the email. Defaults to empty.")
-    @Name("content")
+    @Name(CONTENT)
     @Macro
     @Nullable
     private String content;
 
+    @Name(INCLUDE_WORKFLOW_TOKEN)
+    @Description("Whether to include the contents of the workflow token in the email message. Defaults to false.")
```

Comment:
should we elaborate more on what is a workflow token? since its cdap concept?

Reply:
we absolutely should. I had added it in the markdown file, but forgot to update here. Fixed.